### PR TITLE
Simple VM upgrades via postinstall script

### PR DESCRIPTION
**Pull request by jeffdm**

### CODE REVIEW

File: osx/installerplugins/installerplugins.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -31,17 +23,7 @@
...
-		1B4C91501B8500A700A06068 /* en */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = en; path = en.lproj/Localizable.strings; sourceTree = "<group>"; };
-		1B4C91521B8500A700A06068 /* InstallerSections.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = InstallerSections.plist; sourceTree = "<group>"; };
-		1B4C91571B8500BB00A06068 /* toolboxcheck.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = toolboxcheck.png; sourceTree = "<group>"; };
-		1B4C91581B8500BB00A06068 /* boot2docker.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = boot2docker.png; sourceTree = "<group>"; };
-		1B4C91591B8500BB00A06068 /* toolbox.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = toolbox.png; sourceTree = "<group>"; };
+		1B4C91421B8500A700A06068 /* managevmplugin.bundle */ = {isa = PBXFileReference; explicitFileType = wrapper.cfbundle; includeInIndex = 0; name = managevmplugin.bundle; path = /Users/jmorgan/workspace/toolbox/osx/installerplugins/../mpkg/Plugins/managevmplugin.bundle; sourceTree = "<absolute>"; };
```
Comment (FrenchBen) [inline osx/installerplugins/installerplugins.xcodeproj/project.pbxproj]: Should your user path be here?

File: osx/mpkg/boot2dockeriso.pkg/Scripts/postinstall [hunk 2]
```diff
@@ -2,13 +2,21 @@
 
 CONSOLE_USER=$(stat -f '%Su' /dev/console)
 CONSOLE_USER_HOME=$(su $CONSOLE_USER -c 'echo $HOME')
```
Comment (FrenchBen) [inline osx/mpkg/boot2dockeriso.pkg/Scripts/postinstall]: Looks like this worked before, but if I actually type it in my terminal, I get:
```
su $CONSOLE_USER -c 'echo $HOME'
Password:
```

Comment (jeffdm) [inline osx/mpkg/boot2dockeriso.pkg/Scripts/postinstall]: The installer runs as root so it won't trigger this :smiley:

**Comment by jeffdm** [comment]
@FrenchBen updated! Thanks!

**Comment by FrenchBen** [comment]
LGTM

**Pull request merged**